recipes/py2exe/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "py2exe" %}
+{% set version = "0.11.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/py2exe/py2exe/archive/v{{ version }}.tar.gz
+  sha256: 6bb7ff4834ef2096823710c67edf49c971422775fa41c9c7718c19f0437a2d3d
+
+build:
+  number: 0
+  entry_points:
+    - build_exe = py2exe.build_exe:main
+  skip: True  # [ not win ]
+  skip: True  # [ py < 36 ]
+  
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - setuptools <60
+     # Newer versions cause parsing issues for PY37&38
```

Would suggest this slight tweak. Otherwise looks good :)

```suggestion
     # Newer setuptools versions cause parsing issues for PY37&38
     # xref: https://github.com/py2exe/py2exe/issues/124
    - setuptools <60
```